package.json

```diff
@@ -1,44 +1,44 @@
 {
-  "bin": {
-    "lambda-cfn": "./bin/lambda-cfn.js"
-  },
-  "license": "BSD-2-Clause",
-  "engines": {
-    "node": ">= 6.10.3"
-  },
   "name": "@mapbox/lambda-cfn",
+  "version": "3.0.1",
+  "author": "Mapbox, Inc.",
+  "description": "CloudFormation boilerplate for lambda functions",
+  "license": "BSD-2-Clause",
+  "homepage": "https://github.com/mapbox/lambda-cfn",
   "repository": {
     "url": "https://github.com/mapbox/lambda-cfn.git",
     "type": "git"
   },
-  "author": "Mapbox, Inc.",
-  "homepage": "https://github.com/mapbox/lambda-cfn",
   "bugs": {
     "url": "https://github.com/mapbox/lambda-cfn/issues"
   },
-  "version": "3.0.1",
+  "engines": {
+    "node": ">= 8.10.0"
+  },
   "dependencies": {
-    "aws-sdk": "^2.2.35",
-    "d3-queue": "^2.0.3",
-    "@mapbox/cloudfriend": "^1.9.0",
-    "lodash": "^4.17.4",
+    "@mapbox/cfn-config": "^2.20.1",
+    "@mapbox/cloudfriend": "^3.0.1",
+    "aws-sdk": "^2.474.0",
+    "d3-queue": "^3.0.7",
+    "lodash": "^4.17.11",
```

Comment:
I can't find anywhere we're using this package in the code?

Reply:
Hooray, removing dependencies